lib/web_ui/lib/src/engine/compositor/initialization.dart

```diff
@@ -6,7 +6,7 @@ part of engine;
 
 /// EXPERIMENTAL: Enable the Skia-based rendering backend.
 const bool experimentalUseSkia =
-    bool.fromEnvironment('FLUTTER_WEB_USE_SKIA', defaultValue: false);
+    bool.fromEnvironment('FLUTTER_WEB_USE_SKIA', defaultValue: true);
```

Comment:
How confident are we about the Skia backend?

Reply:
Oops!